resources/public/api/conf/1.0/lossClaims_delete.raml

```diff
@@ -0,0 +1,19 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
+  - formatLossId
```

Comment:
Need FORMAT_LOSS_CLAIM_ID (not FORMAT_LOSS_ID). James' PR has a formatClaimId error trait for this. Maybe merge this in (once in master) and use that.

Reply:
Pulled in master and updated to use formatClaimId